### PR TITLE
Upload for bugfixing

### PR DESCRIPTION
**Pull request by ahohnsen**

### CODE REVIEW

File: src/components/LoadingSpinner.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <span className="sr-only">Loading</span>
+    </SpinnerContainer>
```
Comment (PReichetanz) [inline src/components/LoadingSpinner.js:20]: Whatever it is, I would love to see it in action :D

File: src/pages/BookmarksPage.js [hunk 2]
```diff
@@ -4,31 +4,33 @@ import Content from '../components/Content.js';
...
-                isArchived={wish.isArchived}
-                onToggleBookmark={() => onToggleBookmark(wish.id)}
-                onToggleCheckmark={() => onToggleCheckmark(wish.id)}
-                onEditDiveWish={() => onEditDiveWish(wish)}
-                onDeleteDiveWish={() => onDeleteDiveWish(wish.id)}
-              />
-            ))
+          {bookmarkedPosts?.length > 0 ? (
+            bookmarkedPosts
+              .reverse()
```
Comment (PReichetanz) [inline src/pages/BookmarksPage.js:20]: I looked it up: `.reverse()` mutates the original array [(see here)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse)

**Review by PReichetanz — COMMENTED** [review]
Nice work (regarding the frontend, Lene is checking the backend part) 🚀

**Review by macfrei — COMMENTED** [review]
Nice job :) Added a few comments.

File: src/components/LoadingSpinner.js (hunk 1 above)
Comment (ahohnsen) [reply src/components/LoadingSpinner.js:20]: Checkout the Vercel Link :) it is a loading spinner with lots of dots :)

File: api/posts.mjs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    }
+  } else if (method === 'DELETE') {
+    const { _id } = request.body;
+    try {
+      const deletedPost = await Post.findByIdAndDelete(_id);
+      response.json(deletedPost);
+    } catch (error) {
+      response.status(500).json({ message: error.message });
+    }
+  }
```
[the remark this answers is not in the document]
Comment (ahohnsen) [reply api/posts.mjs:42]: Ok, thank you, I will do that!

File: models/Post.mjs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import mongoose from 'mongoose';
+import './User.mjs';
+const { Schema, model } = mongoose;
+
+const schema = new Schema(
+  {
+    destination: { type: String, required: true },
+    notes: { type: String, required: true },
+    // author: { type: Schema.Types.ObjectId, ref: 'User', required: true },
```
[the remark this answers is not in the document]
Comment (ahohnsen) [reply models/Post.mjs:9]: Oh yes, I had it in there but then decided that I won't need it yet. Forgot to delete the comment. It looked like it worked with that line of code and the `.populate()` method. I will come back to you if I need it in the next few days :)

File: models/Post.mjs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    isArchived: { type: Boolean, required: true },
+    isBookmarked: { type: Boolean, required: true },
```
[the remark this answers is not in the document]
Comment (ahohnsen) [reply models/Post.mjs:11]: Nice! I didn't know that. Thank you

[the remark this answers is not in the document]
Comment (macfrei) [reply models/Post.mjs:11]: You're welcome :)

**Review by mwarnck — APPROVED** [review]

**Pull request merged**